colin/core/ruleset/ruleset.py

```diff
@@ -20,45 +20,45 @@
 
 from six import iteritems
 
-from ..exceptions import ColinConfigException
-from ..constant import CONFIG_DIRECTORY, JSON
+from ..exceptions import ColinRulesetException
+from ..constant import RULESET_DIRECTORY, JSON
 from ..loader import load_check_implementation
 from ..target import is_compatible
 
 logger = logging.getLogger(__name__)
 
 
-class Config(object):
+class Ruleset(object):
 
-    def __init__(self, config_name=None, config_file=None):
+    def __init__(self, ruleset_name=None, ruleset_file=None):
         """
-        Load config for colin.
+        Load ruleset for colin.
 
-        :param config_name: str (name of the config file (without .json), default is "default"
-        :param config_file: fileobj
+        :param ruleset_name: str (name of the ruleset file (without .json), default is "default"
+        :param ruleset_file: fileobj
         """
-        if config_file:
+        if ruleset_file:
             try:
-                logger.debug("Loading configuration from file '{}'.".format(config_file.name))
-                self.config_dict = json.load(config_file)
+                logger.debug("Loading ruleset from file '{}'.".format(ruleset_file.name))
+                self.ruleset_dict = json.load(ruleset_file)
             except Exception as ex:
-                msg = "Config file '{}' cannot be loaded.".format(config_file.name)
+                msg = "Ruleset file '{}' cannot be loaded.".format(ruleset_file.name)
                 logger.error(msg)
-                raise ColinConfigException(msg)
+                raise ColinRulesetException(msg)
         else:
             try:
-                logger.debug("Loading configuration with the name '{}'.".format(config_name))
-                config_path = config_file or get_config_file(config=config_name)
-                with open(config_path, mode='r') as config_file_obj:
-                    self.config_dict = json.load(config_file_obj)
-            except ColinConfigException as ex:
+                logger.debug("Loading ruleset with the name '{}'.".format(ruleset_name))
+                ruleset_path = ruleset_file or get_ruleset_file(ruleset=ruleset_name)
```

Comment:
I think `ruleset_file` is always `None` at this point, because if it was something else we'd not end up in this `else:` branch, but in the above `if ruleset_file:`. What I'm suggesting is
```python
ruleset_path = get_ruleset_file(ruleset=ruleset_name)
```